enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/VolumePermissionEnricher.java

```diff
@@ -175,8 +177,9 @@ public void visit(PersistentVolumeClaimBuilder pvcBuilder) {
                 if (pvcBuilder.buildMetadata() == null) {
                     pvcBuilder.withNewMetadata().endMetadata();
                 }
-                if (!pvcBuilder.buildMetadata().getAnnotations().containsKey(VOLUME_STORAGE_CLASS_ANNOTATION)) {
-                    pvcBuilder.editMetadata().addToAnnotations(VOLUME_STORAGE_CLASS_ANNOTATION, "standard").endMetadata();
+                String storageClass = getConfig(Config.storageClass);
+                if (Strings.isNotBlank(storageClass) && !pvcBuilder.buildMetadata().getAnnotations().containsKey(VOLUME_STORAGE_CLASS_ANNOTATION)) {
```

Comment:
the idea behind storageClass was to define a storage class; happy to rename it defaultStorageClass though (as I guess the annotation in the YAML takes precedence.

Though we want to only add a storage annotation if defaultStorageClass is present with a value - or there's already an annotation in the YAML - we now no longer want to add a 'standard' one